feature-libs/product-configurator/rulebased/components/overview-sidebar/configurator-overview-sidebar.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="cx-sticky-menu">
```

Comment:
@AlexanderKuschner This component is missing a ghost animation. The navigation is visible already when data is still loading